frontend/src/old-pages/Layout.tsx

```diff
@@ -38,6 +39,7 @@ export default function Layout({
         content={children}
         contentType="table"
         navigation={<SideBar />}
+        breadcrumbs={breadcrumbs}
```

Comment:
suggestion: you can centralize implementation of Breadcrumb by putting it here, and just receiving the current breadcrumb "slug"

Then you use your map to:
- find the key for the name of the page, and run it through `i18n`
- find the href (as you are already doing)

Reply:
Did as you said, thanks for the suggestion! Next rev is coming up!